[2024-11-07T18:48:05.407+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-07T18:48:05.460+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SnowflakeTableCreation.create_table scheduled__2024-11-04T06:00:00+00:00 [queued]>
[2024-11-07T18:48:05.478+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SnowflakeTableCreation.create_table scheduled__2024-11-04T06:00:00+00:00 [queued]>
[2024-11-07T18:48:05.480+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-07T18:48:05.505+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2024-11-04 06:00:00+00:00
[2024-11-07T18:48:05.515+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'SnowflakeTableCreation', 'create_table', 'scheduled__2024-11-04T06:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/SnowflakeTableCreation.py', '--cfg-path', '/tmp/tmpatohgftb']
[2024-11-07T18:48:05.519+0000] {standard_task_runner.py:91} INFO - Job 61: Subtask create_table
[2024-11-07T18:48:05.522+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=209) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-07T18:48:05.523+0000] {standard_task_runner.py:63} INFO - Started process 242 to run task
[2024-11-07T18:48:05.594+0000] {task_command.py:426} INFO - Running <TaskInstance: SnowflakeTableCreation.create_table scheduled__2024-11-04T06:00:00+00:00 [running]> on host 4a25c42f06c7
[2024-11-07T18:48:05.777+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='SnowflakeTableCreation' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T06:00:00+00:00'
[2024-11-07T18:48:05.781+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-07T18:48:05.806+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-07T18:48:05.809+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-07T18:48:05.811+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-07T18:48:08.046+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-07T18:48:08.186+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-07T18:48:08.187+0000] {connection.py:762} INFO - closed
[2024-11-07T18:48:08.246+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2024-11-07T18:48:08.308+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-07T18:48:08.309+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-07T18:48:08.317+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=SnowflakeTableCreation, task_id=create_table, run_id=scheduled__2024-11-04T06:00:00+00:00, execution_date=20241104T060000, start_date=20241107T184805, end_date=20241107T184808
[2024-11-07T18:48:08.348+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-07T18:48:08.371+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-07T18:48:08.376+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
